# represents all YOLOv8 detections in one frame

std_msgs/Header header
moiro_interfaces/Detection[] detections